frontend/src/scenes/cohorts/Cohort.tsx

```diff
@@ -210,86 +202,261 @@ export function Cohort({ id }: { id?: CohortType['id'] } = {}): JSX.Element {
                         <Row gutter={24} className="mt">
                             <Col span={24}>
                                 <>
-                                    <div
-                                        className="ant-row ant-form-item ant-form-item-label"
-                                        style={{ display: 'flex' }}
-                                    >
-                                        <label htmlFor="groups" title="Matching criteria">
-                                            Matching criteria
-                                        </label>
-                                        <span>
-                                            Actors who match the following criteria will be part of the cohort.
-                                            Continuously updated automatically.
-                                        </span>
-                                    </div>
-                                    {cohort.groups.map((group: CohortGroupType, index: number) => (
-                                        <Group key={index} name={['groups', index]}>
-                                            <div
-                                                style={{
-                                                    display: 'flex',
-                                                    flexDirection: 'column',
-                                                    alignItems: 'center',
-                                                    width: '100%',
-                                                }}
+                                    <Row align="middle" justify="space-between" wrap={false}>
```

Comment:
I'm not sure if splitting these down into subrender methods will be more readable since they're only being used once in this context and persisting these as variables before the return adds a lot of code in the main method that otherwise won't be rendered in certain contexts. But I see what you mean ternary operators and nested components are hard to read. 

I'll refactor this out into a separate CohortCriteriaGroup component that binds to the same cohortlogic. Hopefully this helps w readability